ajax/libs/bulmaswatch/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "bulmaswatch",
+  "description": "Themes for Bulma",
+  "author": "Jenil Gogari <i@jgog.in>",
+  "homepage": "https://jenil.github.io/bulmaswatch",
+  "filename": "bulmaswatch.min.css",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jenil/bulmaswatch.git"
+  },
+  "license": "MIT",
+  "keywords": [
+    "css",
+    "sass",
+    "flexbox",
+    "responsive",
+    "framework",
+    "themes"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/jenil/bulmaswatch.git",
+    "fileMap": [{
+      "basePath": "",
+      "files": [
+        "bulmaswatch*.+(js|css|map)"
+      ]
+    }]
```

Comment:
Let's let `]` move to next line.
Make same format as others.
the `fileMap` part as the followings:
```js
    "fileMap": [
      {
        "basePath": "",
        "files": [
          "bulmaswatch*.+(js|css|map)"
        ]
      }
    ]
```